tests/core/interfaces-gpio-chardev/mk-sim-chips.sh

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Can you `set -eu` just in case something fails silently.